### PR TITLE
update tags

### PR DESCRIPTION
**Pull request by vierlijner**
Update list of brands for search tags

### CODE REVIEW

**Comment by bobvandevijver** [comment]
This actually needs to be adjusted in the `.homeycompose/app.json` file, from which the `app.json` is being generated. Can you also move the brands on the top in the correct alphabetical order, except for Tuya, Tuya Smart and Smart Life?

**Review by bobvandevijver — CHANGES_REQUESTED** [review]
This actually needs to be adjusted in the .homeycompose/app.json file, from which the app.json is being generated. So both files need an update.

**Comment by vierlijner** [comment]
Done

File: .homeycompose/app.json [hunk 1]
```diff
@@ -76,49 +79,50 @@
...
-      "Grefic",
       "GreenSun",
+      "Grefic",
       "Gremag",
       "GX Diffuser",
       "Hama",
       "HBN",
       "Headerbs",
       "Helloify",
+      "Hema",
```
Comment (bobvandevijver) [inline .homeycompose/app.json:125]:
```suggestion
```

File: app.json [hunk 2]
```diff
@@ -77,49 +80,50 @@
...
-      "Grefic",
       "GreenSun",
+      "Grefic",
       "Gremag",
       "GX Diffuser",
       "Hama",
       "HBN",
       "Headerbs",
       "Helloify",
+      "Hema",
```
Comment (bobvandevijver) [inline app.json:126]:
```suggestion
```

**Review by bobvandevijver — APPROVED** [review]

**Comment by bobvandevijver** [comment]
@vierlijner thanks!

**Pull request merged**